pkg/zson/docs/spec.md

```diff
@@ -278,29 +273,29 @@ sets of strings:
 3:[;];
 ```
 In this example:
-* the first value is a set of one string
-* the second value is a set of two strings "hello" and "world",
-* the third value is an empty set, and
-* the fourth value is a set of one string "".
+* the first value is a `set` of one `string`
+* the second value is a set of two `string` values, `hello` and `world`,
+* the third value is an empty `set`, and
+* the fourth value is a `set` of one zero-length string.
```

Comment:
Perhaps I am being excessively pedantic, but it seems like "" is a representation of how a zero-length string is represented in a language (such as Zeek scripting), not how it's represented in the ZSON data format being described here. Since conceivably different languages may have different ways of representing a zero-length string, here I opted for the verbal description to be generic (I know it's more wordy, though).

Reply:
Proposal: One `string` of zero length.